src/keywords/content.rs

```diff
@@ -5,20 +5,17 @@ use crate::{
     keywords::CompilationResult,
     validator::Validate,
 };
-use serde_json::{from_str, Map, Value};
+use serde_json::{Map, Value};
 
 /// Validator for `contentMediaType` keyword.
 pub(crate) struct ContentMediaTypeValidator {
     media_type: String,
-    func: for<'a> fn(&'a Value, &str) -> ErrorIterator<'a>,
+    func: fn(&str) -> bool,
```

Comment:
What do you think about using `ContentMediaTypeCheckType` here and maybe below when applicable?

Reply:
Totally agree ... I used the lazy `cargo fix` approach ;)